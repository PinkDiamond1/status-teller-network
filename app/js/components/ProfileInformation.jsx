import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { Row, Col } from 'reactstrap';
import Blockies from 'react-blockies';

class ProfileInformation extends Component {
  render() {
    return (
      <Row className="my-5 text-center">
        <Col xs="12">
          <Blockies seed={"denis"} className="rounded-circle"/>
        </Col>
        <Col xs="12">
          <h4 className="font-weight-bold">Denis</h4>
        </Col>
        <Col xs="12">
          <p className="text-muted">0x2376423784623784632784678324</p>
        </Col>
      </Row>
    );
  }
}

ProfileInformation.propTypes = {
};

export default ProfileInformation;
