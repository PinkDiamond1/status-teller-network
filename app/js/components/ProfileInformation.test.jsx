import React from "react";
import { shallow } from "enzyme";

import ProfileInformation from "./ProfileInformation";

describe('ProfileInformation', () => {
  it('should render correctly', () => {
    const component = shallow(<ProfileInformation />);
  
    expect(component).toMatchSnapshot();
  });
});
