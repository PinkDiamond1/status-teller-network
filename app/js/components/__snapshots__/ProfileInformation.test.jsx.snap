// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileInformation should render correctly 1`] = `
<Row
  className="my-5 text-center"
  tag="div"
>
  <Col
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs="12"
  >
    <Identicon
      className="rounded-circle"
      seed="denis"
    />
  </Col>
  <Col
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs="12"
  >
    <h4
      className="font-weight-bold"
    >
      Denis
    </h4>
  </Col>
  <Col
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs="12"
  >
    <p
      className="text-muted"
    >
      0x2376423784623784632784678324
    </p>
  </Col>
</Row>
`;
